malariagen_data/util.py

```diff
@@ -1062,32 +1062,38 @@ def get_gcp_region(details):
                     return "us-west4"
                 elif region == "Texas":
                     return "us-south1"
+                else:
+                    return "us-other"
             elif country == "ZA":
                 if region == "Gauteng":
                     return "africa-south1"
             # Add other regions later if needed.
+            return "other"
```

Comment:
Need to make sure we return a string here, otherwise the location check will be skipped. This was why the warnings were not being raised (#620).